### PR TITLE
Translate devops-user-guide/_index.md

### PR DESCRIPTION
**Pull request by mvpzhangkai**
Signed-off-by: kaige <154599029@qq.com>

### CODE REVIEW

File: content/zh/docs/devops-user-guide/_index.md [hunk 1]
```diff
@@ -1,13 +1,54 @@
...
+linkTitle: "DevOps 用户指南"
 weight: 4400
 
 icon: "/images/docs/docs.svg"
 
 ---
 
-TBD
+
+想要在Kubernetes集群上部署和管理CI/CD任务以及相关的工作负载，请使用KubeSphere DevOps系统。 本章演示了如何在DevOps项目中进行管理和工作，包括运行流水线，创建凭据和集成工具。
```
Comment (FeynmanZhou) [inline content/zh/docs/devops-user-guide/_index.md:14]: 技术文档都有个小规范：中英文或数字之间应该空一格。

Comment (rayzhou2017) [reply content/zh/docs/devops-user-guide/_index.md:14]: I am going to merge and update it. @mvpzhangkai please check my updates when I submit the pr.

**Review by rayzhou2017 — APPROVED** [review]
/lgtm

File: content/zh/docs/devops-user-guide/_index.md (hunk 1 above)
Comment (mvpzhangkai) [reply content/zh/docs/devops-user-guide/_index.md:14]: OK

**Pull request merged**